Automattic/jetpack
# AI Assistant: Remove AI Playground

Fixes #36807

## Proposed changes:

* Removes the AI Playground and its block attributes

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Enable the AI Playground (p1712697233764749/1712697173.113949-slack-C054LN8RNVA)
* Go to the block editor
* Add an AI Assistant block
* Check that the AI Playground is not present
* Test some prompts and actions
* Check that the requests are executed without errors
* Test the extensions, check that the requests are executed without errors
